tgui/packages/tgui/interfaces/MessageMonitor.js

```diff
@@ -0,0 +1,297 @@
+                    />
+                  </Table.Cell>
+                  <Table.Cell>
+                    Shows all messages that have been sent
+                  </Table.Cell>
+                </Table.Row>
+                <Table.Row>
+                  <Table.Cell>
+                    <Button
+                      content={'View Request Console Logs'}
+                      onClick={() => act('view_request_logs')}
+                    />
+                  </Table.Cell>
+                  <Table.Cell>
+                    Shows all orders that were made in the cargo department
+                  </Table.Cell>
+                </Table.Row>
+                <Table.Row>
+                  <Table.Cell>
+                    <Button
+                      content={'Clear Message Logs'}
+                      onClick={() => act('clear_message_logs')}
+                    />
+                  </Table.Cell>
+                  <Table.Cell>Clears message logs</Table.Cell>
+                </Table.Row>
+                <Table.Row>
+                  <Table.Cell>
+                    <Button
+                      content={'Clear Request Console Logs'}
+                      onClick={() => act('clear_request_logs')}
+                    />
+                  </Table.Cell>
+                  <Table.Cell>Clears request console logs</Table.Cell>
+                </Table.Row>
+                <Table.Row>
+                  <Table.Cell>
+                    <Button
+                      content={'Set Custom Key'}
+                      onClick={() => act('set_key')}
+                    />
+                  </Table.Cell>
+                  <Table.Cell>Changes decryption key</Table.Cell>
+                </Table.Row>
+                <Table.Row>
+                  <Table.Cell>
+                    <Button
+                      content={'Send Admin Message'}
+                      onClick={() => act('send_fake_message')}
+                    />
+                  </Table.Cell>
+                  <Table.Cell>
+                    Sends a custom message to the user&apos;s PDA
+                  </Table.Cell>
+                </Table.Row>
+              </>
+            ) : (
+              <Table.Row>
+                <Table.Cell>
+                  <Button
+                    content={'Link Server'}
+                    onClick={() => act('link_server')}
+                  />
+                </Table.Cell>
+                <Table.Cell>Connects to the server</Table.Cell>
+              </Table.Row>
+            )}
+          </Table>
+        </Section>
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+export const MessageMonitor = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { screen, error, succes, notice, server_status } = data;
+  return (
+    <Window width={700} height={400}>
+      <Window.Content>
+        <Stack vertical fill>
+          {server_status === 1 ? (
```

Comment:
These two branches should be separate components, this is really hard to understand.

Reply:
It's a ternary, meaning it has two branches, the true branch and the false branch